edenia/antelope-tools
# Feat/improve ux list endpoints 1203

### Use button with icons instead of only the icon to copy the list of endpoints

### What does this PR do?

- Resolve #1203
- Resolve #1243 
- Fix bug in Accounts and Contracts page, where it search the previous table in contracts without tables

### Steps to test

1. Run the project locally
2. Go to endpoints page
3. Check the text of the buttons
4. Go to nodes page
5. Filter the results by hyperion or history

### Screenshots

## List of endpoints
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/bba2247d-b44f-49e7-a550-df064d44f650)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/a6247a82-c267-4699-a3a1-13a9331d6531)

## Nodes page
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/1da6ed4c-5155-4742-bf06-e5ed414a34d4)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/d43854c2-7d10-4aef-b229-1dc0fdad8e2b)
